homeassistant/components/miele/sensor.py

```diff
@@ -405,7 +408,8 @@ async def async_setup_entry(
                         entity_class = MieleSensor
                 if (
                     definition.description.device_class == SensorDeviceClass.TEMPERATURE
-                    and definition.description.value_fn(device) == -32768 / 100
+                    and definition.description.value_fn(device)
+                    == DISABLED_TEMPERATURE / 100
```

Comment:
I would move the `/100` to the const as well

Reply:
OK, but I planned to use that constant in the climate platform also. There it should not be divided by 100...